Sources/ScipioKit/Package.swift

```diff
@@ -31,24 +30,31 @@ struct Package {
         buildDirectory.appendingPathComponent("\(name).xcodeproj")
     }
 
-    var supportedSDKs: OrderedSet<SDK> {
+    var supportedSDKs: OrderedCollections.OrderedSet<SDK> {
         OrderedSet(manifest.platforms.map(\.platformName).compactMap(SDK.init(platformName:)))
     }
 
+    private static func makeWorkspace(packagePath: AbsolutePath) throws -> Workspace {
+        var workspaceConfiguration: WorkspaceConfiguration = .default
+        // override default configuration to treat XIB files
+        workspaceConfiguration.additionalFileRules = FileRuleDescription.xcbuildFileTypes
+
+        let fileSystem = TSCBasic.localFileSystem
```

Comment:
In this project, there are our `FileSystem` and `TSCBasic.FileSystem`.
I used to consider to drop TSC. However, it was impossible. So there are mixed implementations.
I'll drop `Scipio.FileSystem` in the future.